hubblestack/extmods/modules/nebula_osquery.py

```diff
@@ -241,6 +247,213 @@ def queries(query_group,
     return ret
 
 
+def osqueryd_monitor(servicename='hubble_osqueryd',
```

Comment:
In case of any concerns w.r.t to monitor functionality, contact 'muagarwa@adobe.com'